### PR TITLE
Chromatic for numberfield

### PR DESCRIPTION
**Pull request by snowystinger**
Closes 

## ✅ Pull Request Checklist:

- [ ] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues).
- [ ] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [ ] Filled out test instructions.
- [ ] Updated documentation (if it already exists for this component).
- [ ] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/TR/wai-aria-practices-1.1/)

### CODE REVIEW

File: packages/@react-spectrum/numberfield/chromatic/NumberField.chromatic.tsx [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+export const PropLabelEnd = Template.bind({});
+PropLabelEnd.storyName = 'label end';
+PropLabelEnd.args = {...PropDefaults.args, labelAlign: 'end', defaultValue: 10};
+
+export const PropLabelSide = Template.bind({});
+PropLabelSide.storyName = 'label side';
+PropLabelSide.args = {...PropDefaults.args, labelPosition: 'side', defaultValue: 10};
+
+export const PropCustomWidth = Template.bind({});
```
Comment (ktabors) [inline packages/@react-spectrum/numberfield/chromatic/NumberField.chromatic.tsx]: Should we have stories to test the min-width case in chromatic?

File: packages/@react-spectrum/numberfield/chromatic/NumberField.chromatic.tsx [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+        && !combo.UNSAFE_className.includes('is-focused')
+    );
+    let invalidDisabledState = (
+      combo.isDisabled
+        && combo.UNSAFE_className
+        && combo.UNSAFE_className.includes('is-focused')
+    );
+    return hasClassName && !invalidFocusState && !invalidDisabledState;
+  }
+);
```
Comment (ktabors) [inline packages/@react-spectrum/numberfield/chromatic/NumberField.chromatic.tsx:86]: Should this be commented out or removed?

**Comment by LFDanLu** [comment]
did a sweep of the bugs that we found in the testing session, looks good to me (chrome Android/mac). Will do more thorough testing on tuesday

**Comment by devongovett** [comment]
Looks like the disabled button in quiet issue is still there. Readonly works now though... 🤔 

image

**Review by LFDanLu — COMMENTED** [review]
Overall looks good to me, a couple things with the active/focus state that I noticed in mobile (fine for these to be handled in a followup or dropped if expected behavior):

1. If you hold the increment/decrement button down until hits the maximum/minimum value respectively, the numberfield gets stuck with the focused style. Moving focus to a separate element doesn't seem to remove the focus style but clicking into the input and back out of the input removes the focus style. Screenshot is locally built w/ a `input` element added to the story.
image

2. Tapping the disabled buttons in a readOnly NumberField triggers the number field active style (mobile only).

**Review by devongovett — APPROVED** [review]

**Pull request merged**